### ISSUE TITLE
USWDS - Bug: Banner component creates large CLS when browser enables JavaScript

### ISSUE THREAD

**Issue by mherchel** [uswds/uswds#4928]
### Describe the bug

The [USWDS banner component](https://designsystem.digital.gov/components/banner/) loads in an expanded state. Then when the browser enables JavaScript, it collapses the banner. 

This leads to a large [cumulative layout shift (CLS)](https://web.dev/cls/). 

### Steps to reproduce the bug

This can be troubleshooted using tools like [Pagespeed insights](https://pagespeed.web.dev/) or [WebPageTest](https://www.webpagetest.org/)

![image](https://user-images.githubusercontent.com/1605905/186458946-24426a74-059e-401d-a39e-cf74e03e618c.png)
![image](https://user-images.githubusercontent.com/1605905/186459172-32605847-7c6f-4ee9-8fdc-a152d0071b36.png)

### Expected Behavior

I expect this not to happen, but I still want it to be visible without JavaScript 😬😁

### System setup

USWDS version 2

### Additional context

On SBA.gov, we're solving this by hard coding the banner to be collapsed on pageload 

```scss
.usa-banner__content {
  display: none;

  .usa-banner__header--expanded + & {
    display: block;
  }
}

```
And then we are injecting a `<noscript>` tag into the `<head>` with a CSS styling that tells the browser to make this visible if JavaScript is disabled. 
```css
.usa-banner__content {
  display: block;
}
```

That all being said, there are multiple ways of solving this (I just took an easy route):
1) Use a `<details>` / `<summary>` element. This would be semantically correct, and the best choice 
2) Use something like `focus-within` states on the wrapper surrounding the `<button>`. It'd be a little weird, but it'd work
3) Use the `<noscript>` approach above. 

Anywho, if ya'll give me direction, I'm happy to submit a PR :D 

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by mahoneycm** [comment]
Hey there @mherchel ! Hope you've been well.

This was addressed in #3867 by using `uswds-init.min.js`. I tried replicating the problem but was unable to! I'd give that a lookover and hopefully, it'll help.

**Comment by brunerae** [comment]
Assigning to @thisisdano for closing since this has been fixed.
